.gitignore

```diff
@@ -34,6 +34,7 @@ captures/
 
 # IntelliJ
 *.iml
+.idea/
```

Comment:
Thanks for contribution. your changes are great and they work very fine. I'll provide screens in a while.
This line isn't related to this PR. Please, remove it.

Reply:
I actually mentioned this in the extra context for the PR
This line needs to be added otherwise the .idea files are also added into the commit
To avoid those files, this has to be added

Reply:
@Priyansh-Kedia You can always do `git add src/(...)` instead of `git add .` that adds all changes.
I see however, that another contributor is also adding this.
We can add this on another issue for newcomers

Reply:
@Priyansh-Kedia  did u find a solution for this?

Reply:
I guess we can leave this line here, I'll just approve this.